en/lessons/basics/pipe-operator.md

```diff
@@ -32,14 +32,14 @@ For this set of examples, we will use Elixir's String module.
 - Tokenize String (loosely)
 
 ```elixir
-iex> "Elixir rocks" |> String.split
+iex> "Elixir rocks" |> String.split()
```

Comment:
A quick lookup for `String.split$` has detected a couple more places with stale formatting, would you like to take a stab at those too, @ruslanshakirov?

Reply:
Yes, I'll fix these places.